pom.xml

```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.565.3</version><!-- the minimum Jenkins core compatible version to run plugin on -->
+    <version>1.627</version><!-- the minimum Jenkins core compatible version to run plugin on -->
```

Comment:
It's probably too recent, I'd wait to have a new LTS more recent to merge this. This is mostly to record the minimal version for this to work.